dspace-rest/src/main/java/org/dspace/rest/common/HierarchyRepository.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.rest.common;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Used to handle/determine status of REST API.
+ * Mainly to know your authentication status
+ *
+ */
+@XmlRootElement(name = "repository")
+public class HierarchyRepository extends HierarchyCommunity
```

Comment:
I agree. We should rename `HierarchyRepository` to be `HierarchySite` for clarity, and change anything else to refer to "Site" where it uses the term "Repository". 

It's a tiny change, but it'd be consistent with the fact that we have a "Site" object in DSpace (`org.dspace.content.Site`), but there's no such thing as a "Repository" object.